lua/barbar/events.lua

```diff
@@ -48,7 +48,7 @@ function events.augroups(clear)
   end
 
   return create_augroup('barbar_misc', {clear = clear}),
-    create_augroup('barbar_render', {clear = clear})
+    create_augroup('barbar.ui.render', {clear = clear})
```

Comment:
this intentional?

Reply:
Nope. I did a regex replace for "barbar.render" and technically "_"  the "." 😅